### PR TITLE
feat: update built-in components

### PR DESCRIPTION
**Pull request by trishaprile**
[![PR App][icn]][demo] | Fix RM-10429
:-------------------:|:----------:

## 🧰 Changes
- Update `cards` (linkable, syntax, icon, title), `accordion` (styling, icon, semantic), `tabs` (styling)

| jsx | component |
|--------|--------|
| image | image |
| image | image |
| image | image | 

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: components/index.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
+export { default as Accordion } from './Accordion';
 export { default as Anchor } from './Anchor';
 export { default as Callout } from './Callout';
-export { default as Cards } from './CardsGrid';
+export { default as Cards, Card } from './CardsGrid';
```
Comment (kellyjosephprice) [inline components/index.ts:4]: What about renaming the file to `Cards`?

File: components/Accordion/index.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  const [isActive, setIsActive] = useState(false);
+
+  const handleToggle = () => {
+    setIsActive(!isActive);
+  }
+
+  return (
+    <div className="Accordion">
+      <button className="Accordion-title" onClick={handleToggle}>
+        <i className={`Accordion-arrow${isActive ? '_active' : ''} fa fa-chevron-right`}></i>
```
Comment (kellyjosephprice) [inline components/Accordion/index.tsx:15]: It seems _maybe_ more common to have the toggle icon on the right side? And I think that would be less visualizing confusing with the icon?

Comment (trishaprile) [reply components/Accordion/index.tsx:26]: yea it does look a little odd with the icon.. but i think we had already agreed on the right toggle 😅

**Review by kellyjosephprice — APPROVED** [review]

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v7.2.0</b>](/readmeio/markdown/releases/tag/v7.2.0)
<br>

**Pull request merged**